senza/cli.py

```diff
@@ -217,6 +218,24 @@ def watching(w: bool, watch: int):
 }
 
 
+def decrypt_parameters(definition):
+    senza_info = definition["Mappings"]["Senza"]["Info"]
```

Comment:
This should be something like:

```python
try:
    senza_info = definition["Mappings"]["Senza"]["Info"]
except KeyError:
    return definition
```
to avoid crashing if SenzaInfo is missing.